galaxy_api/api/ui/serializers/namespace.py

```diff
@@ -24,9 +30,9 @@ class Meta:
             'avatar_url',
             'description',
             'links',
+            'groups',
             'resources'
         )
-        read_only_fields = ('name', )
```

Comment:
Won't this make it so namespace owners can change their namespace names?

Reply:
@newswangerd you're right.

Already started working on it in follow-up branch, but this should probably be fixed in here first:

https://github.com/ansible/galaxy-api/compare/master...romanblanco:issue180-ui-update#diff-501a88204e6d0fc39ef5ad9debd9e98dL32-R33

Reply:
- If `'name'` is in `read_only_fields` Meta option, I'm not able to create a namespace (blank name is used):

```
POST /api/automation-hub/v3/_ui/namespaces/ HTTP/1.1
Accept: application/json, */*
Accept-Encoding: gzip, deflate
Connection: keep-alive
Content-Length: 67
Content-Type: application/json
Host: 127.0.0.1:5001
User-Agent: HTTPie/1.0.3

{
    "groups": [
        "demo",
        "system:partner-engineers"
    ],
    "name": "noname"
}

HTTP/1.1 200 OK
Allow: GET, POST, HEAD, OPTIONS
Content-Length: 47
Content-Type: application/json
Date: Fri, 06 Dec 2019 14:03:47 GMT
Server: WSGIServer/0.2 CPython/3.6.8
Vary: Accept
X-Frame-Options: SAMEORIGIN

{
    "success": "Namespace '' created successfully"
}
```

- using `editable` attribute in the model results with the same

```diff
- name = models.CharField(max_length=64, unique=True)
+ name = models.CharField(max_length=64, unique=True, editable=False)
```
- looking if https://docs.djangoproject.com/en/3.0/ref/forms/fields/#disabled could be used.

Reply:
@newswangerd resolved with `extra_kwargs = {'name': {'write_only': True}}`

Reply:
@newswangerd I've used a new serializer `NamespaceUpdateSerializer` to be used while doing `'update'` action.
/cc @cutwater

Reply:
In this case does does `'write_only': True` in this serializer serve any purpose?

Reply:
@cutwater since we're using `read_only_fields = ('name', )` in the serializer for `PUT` request, I think this can be left out. Removing.